src/init.sh

```diff
@@ -27,7 +27,7 @@ function init_kw()
   if ! is_kernel_root "$PWD"; then
     complain 'This command should be run in a kernel tree.'
     if [[ $(ask_yN "Do you want to continue?") =~ '0' ]]; then
-    	exit 125 # ECANCELED
+      exit 125 # ECANCELED
```

Comment:
These problems pointed out by `shfmt` should be fixed before each commit, as to not introduce any commits that are not compliant with our github workflows.